src/main/java/com/exasol/adapter/AdapterFactory.java

```diff
@@ -19,4 +19,11 @@
      * @return new instance
      */
     public VirtualSchemaAdapter createAdapter();
+
+    /**
+     * Get a version of the adapter
```

Comment:
```suggestion
     * Get the version of the Virtual Schema Adapter
```

Reply:
I just realized, that we should also display the newly available adapter version number in a log message. Preferably in the same one where we already show the version number of the common part.